users/views.py

```diff
@@ -1,3 +1,29 @@
 from django.shortcuts import render
+from django.urls import reverse_lazy
+from django.views.generic import (
+    ListView
+)
+from .models import User
 
 # Create your views here.
+class ListarUsuarios(ListView):
+    model = User
+    template_name = "listado.html"
+    context_object_name = "lista_usuario"
+
+    def dispatch(self, request, *args, **kwargs):
+        return super(ListarUsuarios, self).dispatch(request, *args, **kwargs)
```

Comment:
Is ListarUsuarios a method or class?

Reply:
no lo sé copie tal cual estaba en el proyecto

Reply:
Hay 2 tipos de vistas, de clases y de funciones (los metodos son funciones de las clases). 
Si te refieres al super en python se llama con la clase y la ibstancia del objeto

Reply:
Entiendo que los métodos son funciones de las clases (creo que ese no es el problema). Lo que no entiendo es porque esto no esta bien si lo tome del proyecto de referencia así está allá cual debe ser el correcto puede poner codigo para entenderles mejor?